rdkit/ML/AnalyzeComposite.py

```diff
@@ -83,7 +83,7 @@ def ProcessIt(composites, nToConsider=3, verbose=0):
     if verbose >= 0:
       print('# Average Descriptor Positions')
     retVal = []
-    for k in globalRes.keys():
+    for k in sorted(globalRes.keys()):
```

Comment:
The new ordered dictionary by defautl in Python3.6. Alternative would be moving the sort into the test. If sorting is not required, this would be faster. But no issue with it.

The keys() method is not required. 
```
for k in sorted(globalRes):
```